apps/src/code-studio/pd/workshop_dashboard/components/workshop_enrollment_school_info.jsx

```diff
@@ -141,6 +142,16 @@ export class WorkshopEnrollmentSchoolInfo extends React.Component {
     }
   }
 
+  getApplicationURL(application_id, course) {
+    if (!application_id || ![CSD, CSP].includes(course)) {
+      return null;
+    }
+
+    // Note: These paths are defined in ApplicationDashboard component
+    let path = course === CSD ? 'csd_teachers' : 'csp_teachers';
+    return `https://studio.code.org/pd/application_dashboard/${path}/${application_id}`;
```

Comment:
Prefer an origin-relative URL here, which will work in environments other than production:

```suggestion
    return `/pd/application_dashboard/${path}/${application_id}`;
```

Reply:
Fixed